.github/workflows/iso-ci.yaml

```diff
@@ -35,15 +35,16 @@ jobs:
       run: |
         make test
 
-    - name: login to Docker Hub
+    - name: login to Quay Hub
```

Comment:
can you change this to `Quay` or `Quay.io`?  `Hub` is relevant only for Docker Hub

Reply:
fixed